test/include/psen_scan_v2/monitoring_frame_serialization.h

```diff
@@ -15,14 +15,19 @@
 #ifndef PSEN_SCAN_V2_MONITORING_FRAME_SERIALIZATION_H
 #define PSEN_SCAN_V2_MONITORING_FRAME_SERIALIZATION_H
 
-#include "psen_scan_v2/raw_scanner_data.h"
+#include "psen_scan_v2/diagnostics.h"
 #include "psen_scan_v2/monitoring_frame_msg.h"
+#include "psen_scan_v2/monitoring_frame_deserialization.h"
+#include "psen_scan_v2/raw_scanner_data.h"
 
 using namespace psen_scan_v2;
 
 namespace psen_scan_v2
 {
 DynamicSizeRawData serialize(MonitoringFrameMsg& frame);
+std::array<uint8_t, DIAGNOSTIC_DATA_LENGTH_IN_BYTES>
```

Comment:
Maybe create an alias for this array-type because it seems to be used more than once and makes the code hard to read. Maybe something like: 
```
using RawDiagnosticMsg = std::array<uint8_t, DIAGNOSTIC_DATA_LENGTH_IN_BYTES>;
```